src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA");
```

Comment:
I'd create a List to which you can directly put manufacturers object without creation of local variable for each. Then you can iterate the list and create manufacturers. 
```suggestion
        List<Manufacturer> manufacturers = List.of(
                new Manufacturer(null, "Lincoln", "USA"),
                new Manufacturer(null, "Ford", "USA"),
                new Manufacturer(null, "Audi", "Germany"),
                new Manufacturer(null, "BMW", "Germany")
        );

        for (Manufacturer manufacturer : manufacturers) {
            manufacturerDao.create(manufacturer);
        }
```
Next you can create a List of integers with ids of manufacturers which you want to delete and iterate it based on an example above.